src/Octoshift/GithubClient.cs

```diff
@@ -10,10 +11,15 @@ public class GithubClient
         private readonly HttpClient _httpClient;
         private readonly OctoLogger _log;
 
-        public GithubClient(OctoLogger log, HttpClient httpClient)
+        public GithubClient(OctoLogger log, HttpClient httpClient, string personalAccessToken)
         {
             _log = log;
             _httpClient = httpClient;
+
+            _httpClient.DefaultRequestHeaders.Add("Accept", "application/vnd.github.v3+json");
```

Comment:
You might consider wrapping this in an if _httpClient != null block that way you can pass null into the mocks and it won't break